apps/meteor/app/slashcommands-unarchiveroom/server/server.ts

```diff
@@ -9,61 +8,63 @@ import { api } from '../../../server/sdk/api';
 import { roomCoordinator } from '../../../server/lib/rooms/roomCoordinator';
 import { RoomMemberActions } from '../../../definition/IRoomTypeConfig';
 
-function Unarchive(_command: 'unarchive', params: string, item: IMessage): void {
-	let channel = params.trim();
-	let room;
+slashCommands.add(
+	'unarchive',
+	function Unarchive(_command: 'unarchive', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.